test/functional/organisations_controller_test.rb

```diff
@@ -191,7 +191,7 @@ def self.sets_cache_control_max_age_to_time_of_next_scheduled(edition_type)
       organisation_logo_type_id: OrganisationLogoType::CustomLogo.id,
       logo: fixture_file_upload('logo.png')
     )
-    VirusScanHelpers.simulate_virus_scan(organisation.logo)
+
```

Comment:
Minor: Would it be feasible to remove the calls to `VirusScanHelpers.simulate_virus_scan` in a separate commit?